File: system-test/dotnet/SystemTests/Legacy/Mod09/SmokeTests/System/ShopSmokeTest.cs
using SystemTests.Legacy.Mod09.Base;
using Optivem.Testing;
using Xunit;

namespace SystemTests.Legacy.Mod09.SmokeTests.System;

public class ShopSmokeTest : BaseScenarioDslTest
{
    [Theory]
    [ChannelData(ChannelType.UI, ChannelType.API)]
    public async Task ShouldBeAbleToGoToShop(Channel channel)
    {
        await Scenario(channel).Assume().Shop().ShouldBeRunning();
    }
}

File: system-test/java/src/test/java/com/optivem/shop/systemtest/legacy/mod09/smoke/system/ShopSmokeTest.java
package com.optivem.shop.systemtest.legacy.mod09.smoke.system;

import com.optivem.shop.systemtest.legacy.mod09.base.BaseScenarioDslTest;
import com.optivem.shop.testkit.channel.ChannelType;
import com.optivem.testing.Channel;
import org.junit.jupiter.api.TestTemplate;

class ShopSmokeTest extends BaseScenarioDslTest {
    @TestTemplate
    @Channel({ChannelType.UI, ChannelType.API})
    void shouldBeAbleToGoToShop() {
        scenario.assume().shop().shouldBeRunning();
    }
}

